view/theme/frio/templates/paginate.tpl

```diff
@@ -0,0 +1,13 @@
+{{if $pager}}
+<div class="{{$pager.class}}">

```

Comment:
Two different pagers currently can't be on the same page because `paginate_data()` is relying on the current query string and the item count in the App instance.